src/test/java/edu/oakland/test/database/DatabasePersistentStorageStub.java

```diff
@@ -0,0 +1,47 @@
+package edu.oakland.test.database;
+
+import edu.oakland.helper.admin.LocationDataPoint;
+import edu.oakland.helper.admin.TrackData;
+import edu.oakland.production.database.DatabasePersistentStorage;
+
+public class DatabasePersistentStorageStub implements DatabasePersistentStorage {
+  private int rfid;
+  private TrackData trackData;
+  
+  public DatabasePersistentStorageStub(int rfid, TrackData trackdata) {
+    this.rfid = rfid;
+    this.trackData = trackdata;
+  }
```

Comment:
Let's try just doing an RFID for the constructor, as that is easy to set, and TrackData is not.
```suggestion
  public DatabasePersistentStorageStub(int rfid) {
    this.rfid = rfid;
  }
  
```